ProjectLighthouse/Configuration/ConfigurationCategories/CustomizationConfiguration.cs

```diff
@@ -3,4 +3,6 @@ namespace LBPUnion.ProjectLighthouse.Configuration.ConfigurationCategories;
 public class CustomizationConfiguration
 {
     public string ServerName { get; set; } = "Project Lighthouse";
+    public string EnvironmentName { get; set; } = "lighthouse";
```

Comment:
```suggestion
    public string EnvironmentName { get; set; } = "ProjectLighthouse";
```

Reply:
would project-lighthouse suit better?

it's more like official server naming

Reply:
Yeah; should be fine.